recipes/r-class/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
+@rem Add more build steps here, if they are necessary.
+
+@rem See
+@rem http://docs.continuum.io/conda/build.html
+@rem for a list of environment variables that are set during the build process.
```

Comment:
I usually remove these boilerplate comments. In fact I use the `script` line and drop the `bld.bat` and `build.sh` all together like what we do for Python.